src/styles/variables.scss

```
@@ -0,0 +1,15 @@
+$cardWigth: 200px;
+$cadrGapX: 48px;
+$colums: 4;
+$conPad: 40px;
+$mainColor: #616070;
+
+@mixin logoSize {
+  height: 40px;
+  width: 40px;
+}
+
+@mixin starSize {
+  width: 16px;
+  height: 16px;
+}
```

all mixins should be in separate file, not with variables
create one mixin with one argument
```suggestion
@mixin square($size) {
  height: $size;
  width: $size;
}
```